### ISSUE TITLE
No TalkerAlias if using multiple networks

### ISSUE THREAD

**Issue by trancebeyond** [g4klx/DMRGateway#117]
Hello,

So I'm facing an issue with DMRGateway, I'm using BM, TGIF and DMR+, configured routes for each network, however, TA is now not working. Am I missing something?

Here are my rules:

`[DMR Network 1]
Enabled=1
Location=1
Id=xxxxxxxx
PCRewrite1=1,1009990,1,9990,1
PCRewrite2=2,1009990,2,9990,1
TypeRewrite1=1,1009990,1,9990
TypeRewrite2=2,1009990,2,9990
TGRewrite1=1,1000001,1,1,999999
TGRewrite2=2,1000001,2,1,999999
SrcRewrite1=1,9990,1,1009990,1
SrcRewrite2=2,9990,2,1009990,1
SrcRewrite3=1,1,1,1000001,999999
SrcRewrite4=2,1,2,1000001,999999
Address=2262.master.brandmeister.network
Password="xxxxxxx"
Port=62031
Name=BM_2262_Romania

[DMR Network 2]
Enabled=1
Location=0
Id=xxxxxxx
Address=87.255.74.117
Password="PASSWORD"
Port=55555
Name=DMR+_IPSC2-MOLDOVA
PCRewrite1=1,2009990,1,9990,1
PCRewrite2=2,2009990,2,9990,1
TypeRewrite1=1,2009990,1,9990
TypeRewrite2=2,2009990,2,9990
TGRewrite1=1,2000001,1,1,999999
TGRewrite2=2,2000001,2,1,999999
SrcRewrite1=1,9990,1,2009990,1
SrcRewrite2=2,9990,2,2009990,1
SrcRewrite3=1,1,1,2000001,999999
SrcRewrite4=2,1,2,2000001,999999
Debug=0
Options="TS1_1=259;TS2_1=2599;"

[DMR Network 3]
Enabled=0
Name=HBLink
Address=1.2.3.4
Port=5555
TGRewrite0=2,11,2,11,1
Password="PASSWORD"
Location=0
Debug=0

[DMR Network 4]
Enabled=1
Name=TGIF_Network
PCRewrite1=1,4009990,1,9990,1
PCRewrite2=2,4009990,2,9990,1
TypeRewrite1=1,4009990,1,9990
TypeRewrite2=2,4009990,2,9990
TGRewrite1=1,4000001,1,1,999999
TGRewrite2=2,4000001,2,1,999999
SrcRewrite1=1,9990,1,4009990,1
SrcRewrite2=2,9990,2,4009990,1
SrcRewrite3=1,1,1,4000001,999999
SrcRewrite4=2,1,2,4000001,999999
Address=tgif.network
Password=xxxxxxx
Port=62031
Location=0
Debug=0
Id=xxxxxxx`

Thanks in advance

**Comment by W0CHP** [comment]
IIRC, DMR TA is ultimately handled by MMDVMHost  (@g4klx  can correct me if I'm wrong).

The following lines in the MMDVMHost INI config file control TA:

```ini
[DMR]
EmbeddedLCOnly=0
DumpTAData=1
```
Also, not all networks pass/support TA (read: DMR+/MARC)

**Comment by trancebeyond** [comment]
Yeah I know, those 2 options are in place. And I'm aware about TA in DMR+, the thing is that even BM network stops parsing TA after multiple network configuration

**Comment by kantooon** [comment]
I think this change: https://github.com/g4klx/DMRGateway/pull/127 should solve the issue. It only occurs during rewrite, so if a PassAllTG or PassAllPC is hit first the issue will not manifest.
